vxyowsup/whatsapp.py

```diff
@@ -39,3 +58,51 @@ def handle_outbound_message(self, message):
             return self.publish_nack(message['message_id'], 'failed')
         return self.publish_ack(message['message_id'], 
             'remote-message-id')
+            
+            
+    def print_error(self, f):
+        print f
+        return f
+            
+class Client:
```

Comment:
This should probably inherit from `object` (for backwards compatibility with ancient versions of Python, classes that don't inherit from anything in Python 2 are rather odd).